cookbooks/bcpc-hadoop/recipes/hdfs_user_directories.rb

```diff
@@ -0,0 +1,55 @@
+require "mixlib/shellout"
+Chef::Resource::RubyBlock.send(:include, Bcpc_Hadoop::Helper)
+
+%w(ldap-utils libsasl2-modules-gssapi-mit).each do |pkg|
+  package pkg do
+    action :upgrade
```

Comment:
Other recipes did `install` instead of `upgrade`. Is there a standard. I believe `upgrade` was what we always did before.